pkg/kubelet/lifecycle/predicate.go

```diff
@@ -223,6 +225,12 @@ func (e *PredicateFailureError) GetReason() string {
 
 // GeneralPredicates checks a group of predicates that the kubelet cares about.
 func GeneralPredicates(pod *v1.Pod, nodeInfo *schedulerframework.NodeInfo) ([]PredicateFailureReason, error) {
+	return CheckPodOnNode(pod, nodeInfo)
```

Comment:
As I mentioned above, I'd expect to reuse the `GeneralPredicates` (can be renamed to `GeneralFilter`) in the scheduler package.

Reply:
done.